frontend/app/components/st-time-slot-form/st-time-slot-form.tsx

```diff
@@ -115,13 +115,13 @@ export function StTimeSlotForm(props: StTimeSlotFormProps) {
   // update times when timeMode changes
   React.useEffect(() => {
     setLoading(true);
-    let mounted = true;
+    let mounted = true
```

Comment:
Did you add this to stop the error `Warning: Can't perform a React state update on an unmounted component`

Reply:
yes , do you know another way to fix it? this is what i found on google

Reply:
No its a good fix I think, not sure how it works but seems to work fine.